journal-of-american-mosquito-control-association.csl

```diff
@@ -0,0 +1,261 @@
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true">
+    <sort>
+      <key macro="year-date"/>
+      <key macro="author-short"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group delimiter=" ">
+        <text macro="author-short"/>
+        <text macro="date"/>
+      </group>
+      <text variable="locator" prefix=": "/>
+    </layout>
+  </citation>
+  <bibliography hanging-indent="false" et-al-min="11" et-al-use-first="10">
+    <sort>
+      <key macro="author"/>
+      <key macro="year-date"/>
+    </sort>
+    <layout suffix=".">
+      <group suffix="." delimiter=". ">
+        <text macro="author"/>
+        <text macro="date"/>
+        <text macro="title"/>
+      </group>
+      <group suffix=".">
+        <choose>
+          <if type="bill book graphic legal_case legislation motion_picture report song thesis" match="any">
+            <group prefix=" " suffix="." delimiter=" ">
+              <text macro="edition"/>
+              <text macro="editor"/>
+            </group>
+            <text prefix=" " macro="publisher"/>
+            <text prefix=" " macro="collection"/>
+          </if>
+          <else-if type="chapter paper-conference entry-dictionary entry-encyclopedia" match="any">
+            <group prefix=" " delimiter=" ">
+              <group delimiter=" ">
+                <text term="in" text-case="capitalize-first" suffix=":"/>
+                <text macro="editor"/>
+                <text variable="container-title" suffix="."/>
+              </group>
+              <text variable="volume" prefix="Vol. " suffix="."/>
+              <text macro="edition"/>
+              <group suffix="." delimiter=". ">
+                <text macro="publisher"/>
+                <text prefix=" " macro="collection"/>
+                <text macro="pages"/>
+              </group>
+            </group>
+          </else-if>
+          <else-if type="review review-book" match="any">
+            <text macro="editor" prefix=" " suffix="."/>
+            <group prefix=" ">
+              <text macro="review" suffix="."/>
+              <group prefix=" ">
+                <text variable="volume"/>
+                <text variable="issue" prefix="(" suffix=")"/>
+              </group>
+              <text variable="page" prefix=":"/>
+            </group>
+          </else-if>
+          <else>
+            <text macro="editor" suffix="." prefix=" "/>
+            <group prefix=" ">
+              <text variable="container-title" form="short" strip-periods="true" font-style="italic" suffix="."/>
+              <text variable="volume" prefix=" "/>
+              <text variable="issue" prefix="(" suffix=")"/>
+              <text variable="page" prefix=":"/>
+            </group>
+            <text variable="DOI" prefix=". doi:"/>
```

Comment:
Use `https://doi.org/` as the DOI prefix, rather than `doi: `. This is the preferred format, and it is used this way in several articles in the journal I checked.